sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2573,20 +2568,22 @@
       }
     },
     "TwinProperties": {
-      "description": "Represents Twin properties",
+      "description": "Represents twin's desired and reported properties. The maximum depth of twin property objects is 10.",
       "type": "object",
       "properties": {
         "desired": {
-          "description": "Used in conjunction with reported properties to synchronize device configuration or condition. Desired properties can only be set by the solution back end and can be read by the device app. The device app can also be notified in real time of changes on the desired properties.",
+          "description": "Properties that set by the solution back end and read by the device.",
```

Comment:
"... that **are** set by the solution..."

Reply:
These will be readonly fields, so I'm not sure how useful this description is

Reply:
What is solution here? Service? I did not understand.

Reply:
These are not all readonly fields, this is how the service client would update desired properties. For reported properties, yes, it would be readonly.

Reply:
"Properties that **are** set by"

Reply:
On a different note - Should the details about who can update this be on the API that does the update and not the object? That is just a separate thing to think about but keep this description for now.

Reply:
The `TwinProperties` object isn't used by an API exclusively. It is embedded in a `Twin`, which has other fields as well. So I can't find a good home for this description anywhere else.